intelmq/bots/collectors/rsync/collector_rsync.py

```diff
@@ -23,20 +42,73 @@ def init(self):
         except FileExistsError:
             pass
 
+        if self.extra_params:
+            self.extra_params = shlex.split(self.extra_params)
+        else:
+            self.extra_params = []
+
+        if self.private_key and self.private_key_path:
+            raise Exception("You must define only one of the variables private_key or private_key_path")
+
+        if self.private_key:
+            bot_id = f'{self!r}'.split('.')[-1].split(' ')[0] # TODO: use bot id
```

Comment:
Are you looking for `self._Bot__bot_id`?

Reply:
right, thanks